cclib/parser/molcasparser.py

```diff
@@ -696,6 +699,32 @@ def extract(self, inputfile, line):
                     self.mocoeffs = []
                 self.mocoeffs.append(mocoeffs)
 
+        # Parsing data ccenergies from &CCSDT module.
+        #  ()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()()
+        #
+        #                                                 &CCSDT        
+        # ...
+        # ...
+        #          14          -75.01515915      -0.05070274      -0.00000029
+        #          15          -75.01515929      -0.05070289      -0.00000014
+        #          16          -75.01515936      -0.05070296      -0.00000007
+        #       Convergence after                    17  Iterations
+        #
+        #
+        #      Total energy (diff) :     -75.01515936      -0.00000007
+        #      Correlation energy  :        -0.0507029554992
+        if line.startswith('()'):
```

Comment:
Is it possible to trigger on the text in the previous line instead? That seems more appropriate and clearly related to CC.

Reply:
Yeah. That way we can computation since all modules print `()()()`.